packages/core/admin/admin/src/pages/UseCasePage/index.js

```diff
@@ -77,22 +76,21 @@ const UseCasePage = () => {
   const { firstname, email } = auth.getUserInfo();
   const { hasAdmin } = parse(location?.search, { ignoreQueryPrefix: true });
   const isOther = role === 'other';
+  const { post } = useFetchClient();
```

Comment:
Could you please move this up where the other hooks are being called? We usually do this in case someone needs to introduce an early return, which would cause hooks that are listed at the end to not being called and therefore cause a react error.

Reply:
yes sure, thanks

Reply:
done